content/en/security/vulnerabilities/troubleshooting.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Troubleshooting Cloud Security Management Vulnerabilities
+kind: documentation
+further_reading:
+- link: "/security/cloud_security_management/setup/csm_pro/?tab=aws#configure-the-agent-for-containers"
+  tag: "Documentation"
+  text: "Setting up container image vulnerabilities"
+- link: "/security/cloud_security_management/setup/csm_enterprise?tab=aws#configure-the-agent-for-vulnerabilities"
```

Comment:
Noticed this opens up the page to the Kubernetes tab, should this be for the host tab?
```suggestion
- link: "/security/cloud_security_management/setup/csm_enterprise/?tab=hosts#configure-the-agent-for-vulnerabilities"
```

Reply:
Thanks, you are correct! I will fix it.